langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -218,6 +238,86 @@ void test_extract_enum_with_description() {
                 "SNOWY - Snowfall occurs, covering the ground in white and creating cold, wintry conditions")));
     }
 
+    public enum Ingredient {
+        SALT,
+        PEPPER,
+        VINEGAR,
+        OIL
+    }
+
+    interface IngredientsExtractor {
+
+        @UserMessage("Analyze the following recipe:\n|||{{it}}|||")
+        List<Ingredient> extractIngredients(String recipe);
+    }
+
+    @Test
+    void test_extract_list_of_enums() {
+        IngredientsExtractor ingredientsExtractor = AiServices.create(IngredientsExtractor.class, chatLanguageModel);
+
+        String recipe = "Just mix some salt, pepper and oil in the bowl. That will be a basis for...";
+
+        List<Ingredient> ingredients = ingredientsExtractor.extractIngredients(recipe);
+        System.out.println(ingredients);
```

Comment:
minor: please add an assertion on `ingredients` to make sure LLM understands the prompt